server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -192,6 +193,41 @@ public CompletableFuture<Token> updateToken(ServiceRequestContext ctx,
         );
     }
 
+    /**
+     * PATCH /tokens/{appId}/permission
+     *
+     * <p>Updates a permission of a token of the specified ID.
+     */
+    @Patch("/tokens/{appId}/level")
+    @RequiresAdministrator
+    public CompletableFuture<Token> updateTokenLevel(ServiceRequestContext ctx,
+                                                     @Param String appId,
+                                                     TokenLevelRequest tokenLevelRequest,
+                                                     Author author, User loginUser) {
+
+        return getTokenOrRespondForbidden(ctx, appId, loginUser).thenCompose(
+                token -> {
+                    switch (tokenLevelRequest.getLevel()) {
```

Comment:
What do you think about supporting case-insensitive requests?
```suggestion
                    switch (tokenLevelRequest.getLevel().toLowerCase()) {
```